sdk/formrecognizer/Azure.AI.FormRecognizer/src/ExtractedLabeledForm.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+using Azure.AI.FormRecognizer.Models;
+
+namespace Azure.AI.FormRecognizer.Custom
+{
+    /// <summary>
+    /// </summary>
+    public class ExtractedLabeledForm
+    {
+        internal ExtractedLabeledForm(DocumentResult_internal documentResult, IList<PageResult_internal> pageResults, IList<ReadResult_internal> readResults)
+        {
+            // Supervised
+            FormType = documentResult.DocType;
+
+            // TODO: validate that PageRange.Length == 2.
+            // https://github.com/Azure/azure-sdk-for-net/issues/10547
+            StartPageNumber = documentResult.PageRange.First();
+            EndPageNumber = documentResult.PageRange.Last();
+
+            Fields = ConvertFields(documentResult.Fields, readResults);
+
+            Tables = ConvertLabeledTables(pageResults, readResults);
+
+            if (readResults != null)
+            {
+                RawExtractedPages = ConvertRawPages(readResults);
+            }
+        }
+
+        /// <summary>
+        /// </summary>
+        public string FormType { get; internal set; }
+
+        /// <summary>
+        /// </summary>
+        public int StartPageNumber { get; internal set; }
+
+        /// <summary>
+        /// </summary>
+        public int EndPageNumber { get; internal set; }
+
+        /// <summary>
+        /// </summary>
+        public IReadOnlyList<ExtractedLabeledField> Fields { get; }
```

Comment:
I think Johan already asked this, but if the keys are unique, why not a dictionary instead of a list here?

Reply:
This is left over from the merge of supervised and unsupervised fields, since for unsupervised the keys are not unique.  I'd prefer to keep these two consistent as much as we can.